service/src/java/org/apache/hive/service/cli/thrift/ThriftHttpServlet.java

```diff
@@ -551,7 +551,7 @@ String doKerberosAuth(HttpServletRequest request)
       throws HttpAuthenticationException {
     // Each http request must have an Authorization header
     // Check before trying to do kerberos authentication twice
-    getAuthHeader(request, authType);
+    getAuthHeader(request, "NEGOTIATE");
```

Comment:
I meant to ask about this earlier. For kerberos auth, if the header contained "NEGOTIATE", how did this work earlier when we were looking for header named "KERBEROS" ?

Reply:
This argument is unused, so it should be safe to remove it entirely.

Reply:
makes sense. Please mark this comment as resolved.